htdocs/language/english/locale.php

```
@@ -63,14 +63,14 @@ public function number_format($number)
      *
-     * @param  string $format
-     * @param  string $number
-     * @return money  format
+     * @param  string    $format  sprintf-style format (ignored, kept for BC)
+     * @param  int|float $number
+     * @return string
      */
     public function money_format($format, $number)
     {
-        setlocale(LC_MONETARY, 'en_US');
+        $fmt = new \NumberFormatter('en_US', \NumberFormatter::CURRENCY);
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '72', column '20'). (undefined)

(MissingImport)